src/components/MyWorkOrders.js

```diff
@@ -17,15 +18,17 @@ class MyWorkOrders extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      myWorkOrdersData: []
+      myWorkOrdersData: [],
+      loading: false
     };
   }
   componentDidMount() {
+    this.setState({ loading: true });
```

Comment:
One suggestion that we mentioned verbally is to let `loading` be true by default to avoid an extra (I think unneeded state change).